src/prototype_room_creepbuilder.js

```diff
@@ -321,17 +316,42 @@ Room.prototype.spawnCreateCreep = function(creep) {
     buildRoad: unit.buildRoad,
```

Comment:
I think that line is unneeded anymore, because the access to the role settings is easier. I think I introduced it, so that I don't have to require the old old `creep_` files, to check if a role is build.

Don't want to touch it in this PR.